### DISCUSSION TITLE
htmo/3.0.0/use_HTMO_mail_reqs

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#158] [Q&A]
# htmo/3.0.0/use_HTMO_mail_reqs

Make sure that your computer complies with the system requirements to use HCL Traveler for Microsoft Outlook. In addition, make sure that you set required settings in Microsoft Outlook and HCL Notes.

https://help.hcl-software.com/htmo/3.0.0/use_HTMO_mail_reqs.html

**Comment by Davehabz** [comment]
I think you can delete the column for SmartCloud Notes.  That has been gone since the time HCL took over Notes, circa version 11.
